open-craft/xblock-poll
# [BB-3673] Recompile translations and cleanup translations folder

Recompile translations and cleanup translations folder.
This fixes an issue with some case insensitive filesystems, which made it impossible to clone the repo.

See https://openedx.slack.com/archives/C01ATGRBBDM/p1612298099047900 for context.

**Testing instructions:**
1. Install block.
2. Check that translations are working as expected.
3. Check that there are no case-insensitive name conflicts in the translation folder.

**Not sure if removing the files broke the translations** (all new ones where automatically generated).

**Reviewer:**
- [x] @bradenmacdonald Since you use a mac :)
- [x] @kaizoku As FF.
